api/package.json

```diff
@@ -54,6 +60,7 @@
     "@tekuconcept/nestjs-csrf": "^1.1.0",
     "bcryptjs": "^2.4.3",
     "cache-manager": "^5.3.2",
+    "class-transformer": "^0.5.1",
```

Comment:
@yassinedorbozgithub I'm not sure why I got the error that this package was missing ...

Reply:
Hi @marrouchi, maybe it's related to the IDE bug. 
I got something similar before and it's resolved by restarting the IDE.